src/main/kotlin/lotto/domain/LottoTickets.kt

```diff
@@ -2,15 +2,22 @@ package lotto.domain
 
 import lotto.dto.WinStats
 import lotto.util.AutoNumbers
+import lotto.util.RandomNumbers
 
-class LottoTickets(private val money: Money, autoNumbers: AutoNumbers) {
-    val lottoTickets: List<LottoTicket> = List(money.countLotto()) {
-        LottoTicket(autoNumbers.generateNumbers())
+class LottoTickets(private val money: Money) {
```

Comment:
lottoTickets를 만들어주는걸 밖에서 해주면 어떨까요?
아래와 같이 주생성자를 변경해보는것도 좋을 것 같아요!!

```suggestion
class LottoTickets(private val money: Money, val lottoTickets: List<LottoTicket>) {
```